### DISCUSSION TITLE
v8 and React 18?

### DISCUSSION THREAD

**Discussion by blendsdk** [microsoft/fluentui#26582] [Q&A] [answered]
Hi,

Can we use React 18 with v8 or should we continue with react 17?

Thanks.

**Comment by layershifter** [comment, marked as answer]
@blendsdk React 18 is allowed in `peerDependencies` for `@fluentui/react`, so you can use it 😉 

https://github.com/microsoft/fluentui/blob/e2458d07be3e2c21bf2ca5e17495a4204fe06441/packages/react/package.json#L68-L69

**Reply by blendsdk** [reply]
Thank you
